cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -134,10 +110,15 @@ users:
 		expectedError    string
 	}{
 		{
-			name:             "config file",
-			options:          &Options{ConfigFile: configFile},
+			name:             "config file in kubescheduler.config.k8s.io/v1alpha1",
+			options:          &Options{ConfigFile: configFile1},
 			expectedUsername: "config",
 		},
+		{
+			name:          "config file in componentconfig/v1alpha1",
+			options:       &Options{ConfigFile: configFile2},
+			expectedError: "no kind \"KubeSchedulerConfiguration\" ",
```

Comment:
could we improve this error message by improving the function to first only load typemeta and error with a good message if it is `componentconfig/v1alpha1` with an action item to migrate?